openfecwebapp/templates/macros/filters/date.html

```diff
@@ -46,7 +46,6 @@
   </div>
   <div class="filter js-filter" id="{{ name }}" data-filter="date" data-name="{{ name }}" data-validate="true">
     <fieldset>
-    <legend class="label">{{ label }}</legend>
```

Comment:
I don't think we need the `legend` or `fieldset`. `fieldset` is just for grouping form elements that share the same name (like a group of checkboxes or radio buttons). So we don't need it here. And `legend` is only necessary to label a `fieldset`, so we don't need it either.